tpch/tpchdata.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:22.04
+
+RUN apt-get update && \
+    apt-get install -y git build-essential
+
+# Use https://github.com/databricks/tpch-dbgen to generate data
+RUN git clone https://github.com/databricks/tpch-dbgen.git && cd tpch-dbgen && make
```

Comment:
Will it generate parquet files?

Reply:
Ok, I thought databricks has supported generate parquet data, but it seems they didn't...